### PR TITLE
Localized logo and small style improvements

### PR DESCRIPTION
**Pull request by Macroz**
- Remove extra padding from event bubbles
- Remove some extra divs
- Improve localization by allowing a logo per language.
- See also https://github.com/CSCfi/rems-deploy/pull/18
- See #872

### CODE REVIEW

File: src/clj/rems/css/styles.clj [hunk 1]
```diff
@@ -11,8 +11,23 @@
...
+
+(defn resolve-image [path]
+  (when path
+    (let [url (if (str/starts-with? path "http")
+                path
+                (str (util/get-theme-attribute :img-path "../img/") path))]
+      (str "url(\"" url "\")"))))
+
+(defn get-logo-image [lang]
+  (prn :kieli lang)
```
Comment (opqdonut) [inline src/clj/rems/css/styles.clj]: stray prn

File: src/clj/rems/css/styles.clj [hunk 2]
```diff
@@ -395,7 +403,7 @@
             :margin-bottom (u/em 1)}]
    [(s/descendant :.logo :.img) {:height "100%"
                                  :background-color (util/get-theme-attribute :logo-bgcolor)
-                                 :background-image (resolve-image (util/get-theme-attribute :logo-name))
+                                 :background-image (get-logo-image context/*lang*)
```
Comment (opqdonut) [inline src/clj/rems/css/styles.clj]: should there also be a fallback when the language-specific image isn't defined? or YAGNI?

File: src/cljs/rems/application.cljs [hunk 3]
```diff
@@ -565,21 +565,19 @@
...
-(defn- event-groups-view [event-groups]
-  [:div
-   (into [:div]
-         (for [group event-groups]
-           ^{:key group} [:div.group
-                          (for [e group]
-                            ^{:key e} [event-view e])]))])
+(defn- render-event-groups [event-groups]
+  (for [group event-groups]
+    ^{:key group} [:div.group
```
Comment (foxlynx) [inline src/cljs/rems/application.cljs]: Are the two levels of div from events-groups-view dropped on purpose? 
```
(defn- event-groups-view [event-groups]
  [:div
   (into [:div]
         (for [group event-groups]
```

File: src/clj/rems/css/styles.clj (hunk 2 above)
Comment (Macroz) [reply src/clj/rems/css/styles.clj]: The fallback is there, but it's actually in the `get-logo-image` fn.

**Pull request merged**